src/util/ArraysEqual.js

```diff
@@ -2,13 +2,5 @@ module.exports = function arraysEqual(a, b) {
   if (a === b) return true;
   if (a.length !== b.length) return false;
 
-  for (const itemInd in a) {
-    const item = a[itemInd];
-    const ind = b.indexOf(item);
-    if (ind) {
-      b.splice(ind, 1);
-    }
-  }
-
-  return b.length === 0;
+  return a.every(e => b.includes(e));
```

Comment:
Neither does the previous one, also the previous one wouldn't remove any item at index 0 (if-statement), but will run `b.splice(-1, 1)` when the item is not found.

Unless there's some black magic going on that I don't understand.